pkg/build.ml

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env ocaml 
+#directory "pkg"
+#use "topkg.ml"
+
+let () = 
+  Pkg.describe "higher" ~builder:(`OCamlbuild []) ([
+      Pkg.lib "pkg/META";
+      Pkg.lib ~exts:Exts.module_library "higher";
+      Pkg.doc "README.md" ]
+      @ List.map (fun e -> Pkg.share ("examples/"^e))
+        (Array.to_list (Sys.readdir "examples")))
```

Comment:
I usually list them explicitly, but why not... (I personally put sample code in `Pkg.doc` but that may be an error on my side).